lib/tasks/enrichManifest.js

```diff
@@ -0,0 +1,41 @@
+import manifestEnricher from "../processors/manifestEnricher.js";
+import fsInterface from "@ui5/fs/fsInterface";
+import {getLogger} from "@ui5/logger";
+const log = getLogger("builder:task:enrichManifest");
+
+/* eslint "jsdoc/check-param-names": ["error", {"disableExtraPropertyReporting":true}] */
+/**
+ * Task for transforming the manifest.json file.
+ *
+ * @public
+ * @function default
+ * @static
+ *
+ * @param {object} parameters Parameters
+ * @param {@ui5/fs/DuplexCollection} parameters.workspace DuplexCollection to read and write files
+ * @param {object} parameters.options Options
+ * @param {string} parameters.options.projectNamespace Namespace of the application
+ * @returns {Promise<undefined>} Promise resolving with <code>undefined</code> once data has been written
+ */
+export default async function({workspace, options}) {
+	// Backward compatibility: "namespace" option got renamed to "projectNamespace"
```

Comment:
Do we have to add this to a new task? I would hope that it is only relevant for existing tasks.

Reply:
removed